app/views/profile/index.html.slim

```diff
@@ -14,50 +13,59 @@ h1.hide = t('titles.profile')
 
 = render @view_model.header_partial, view_model: @view_model
 
-= render @view_model.pii_partial, decrypted_pii: @view_model.decrypted_pii
-
-.mt2.mb4
-  .py-12p.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-5.px1 = t('.email')
-      .sm-col.sm-col-5.px1.truncate = current_user.email
-      .sm-col.sm-col-2.px1.sm-right-align
-        = link_to manage_email_path, class: btn_cls do
-          span.hide = t('.email')
-          = t('forms.buttons.edit')
-  .py-12p.border-top.border-bottom
-    .clearfix.mxn1
-      .sm-col.sm-col-10.px1 = t('.password')
-      .sm-col.sm-col-2.px1.sm-right-align
-        = link_to manage_password_path, class: btn_cls do
-          span.hide = t('.password')
-          = t('forms.buttons.edit')
-  .mt3.mb1.h6.bold.caps.ls-0
-    | #{t('headings.profile.two_factor')}#{tooltip(t('tooltips.two_factor'))}
-  .py-12p.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-5.px1 = t('.phone')
-      .sm-col.sm-col-5.px1 = current_user.phone
-      .sm-col.sm-col-2.px1.sm-right-align
-        = link_to manage_phone_path, class: btn_cls do
-          span.hide = t('.phone')
-          = t('forms.buttons.edit')
-  .py-12p.border-top
-    .clearfix.mxn1
-      .sm-col.sm-col-10.px1 = t('.authentication_app')
-      .sm-col.sm-col-2.px1.sm-right-align
-        = render @view_model.totp_partial, btn_cls: btn_cls
-
-  = render @view_model.manage_personal_key_partial, btn_cls: btn_cls
-
-h2.h3.my0 = t('headings.profile.account_history')
-.mt2.mb2
-  - decorated_user = current_user.decorate
-  - events_len = decorated_user.recent_events.length
-  - decorated_user.recent_events.each_with_index do |event, i|
-    div class="py-12p border-top #{'border-bottom' if i + 1 == events_len}"
-      .clearfix.mxn1
-        .sm-col.sm-col-6.px1
-          .truncate = event.pretty_event_type
-        .sm-col.sm-col-6.px1.sm-right-align
-          = event.happened_at
+.profile-info-box
+  .bg-blue-light.py1.px2.h6.caps
+    = t('profile.index.login')
+    = image_tag asset_url('sign-in.svg'), width: 12, class: 'ml1 mt-tiny'
+  = render 'profile_item',
+    name: t('profile.index.email'),
+    content: current_user.email,
+    path: manage_email_path,
+    action: @view_model.edit_action_partial
+  .border-bottom.border-blue-light
```

Comment:
Could this line be added to the containing `.p2.clearfix` of `_profile_item.html.slim` instead of having the extra div?